.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+on:
+  release:
+    branches:
+      - main
+    types: [created]
+name: Publish release
+jobs:
+  release_linux:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install build tools
+        run: sudo apt-get update; DEBIAN_FRONTEND="noninteractive" sudo apt-get -y install build-essential curl tzdata 
+      - name: Install openssl
+        run: sudo apt install -y libssl-dev pkg-config
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+      - uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --release
+      - name: Rename
+        run: mv ./target/release/drg ./target/release/drg-linux
+      - uses: AButler/upload-release-assets@v2.0
```

Comment:
You can use the "hub" tool to do the same:

https://github.com/drogue-iot/drogue-cloud/blob/77c264c5bd19bf0005194de44e828d23b1594dd1/.github/workflows/ci.yaml#L255-L276

Reply:
there are some actions as well https://github.com/softprops/action-gh-release

Reply:
I made changes, now we use `hub` tool

Previously, I used this action softprops/action-gh-release but it has some issues which need a workaround, thus switched to other methods.